manager/testdata/new-blueprints/scenario2/plotter.yaml

```diff
@@ -136,10 +118,22 @@ spec:
       kind: M4DModule
       chart:
         name: ghcr.io/mesh-for-data/read-module:0.1.0
+      capabilities:
+        - capability: read
+          scope: workload
     - name: transform-conf
       kind: M4DModule
       chart:
         name: ghcr.io/mesh-for-data/m4d-arrow-flight-transform-conf:0.1.0
+      capabilities:
```

Comment:
Why do we need to add the capabilities here? Capabilities are used in the module decision process which is finished at this stage because this is the output of it.

Reply:
We need to know the scope in order to determine the number of instances to deploy. We need to know the context (this module is used for read although it can also write) in order to construct the arguments: source for the read module, sink for the write, etc.

Reply:
If we're using this to choose which capability of a module to use can this field be a struct and not an array?

Reply:
Yes, it should be a struct.

Reply:
if a chart has multiple capabilities and different capabilities are used in different steps then the chart will appear several times each time with the capability that the step needs, is that correct?

Reply:
Yes